__tests__/app.test.js

```diff
@@ -182,6 +183,75 @@ describe("CORE: GET - /api/articles/:article_id", () => {
             expect(dates).toBeSorted({ descending: true })
           });
       });
+      test("200: Topic query within endpoint returns articles filtered by specific topic", () => {
+        return request(app)
+          .get("/api/articles?topic=mitch")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).not.toBe(0)
+            body.articles.forEach((article) => {
+                expect(article).toHaveProperty("topic", ('mitch'))
+            })        
+          });
+      });
+      test("200: Invalid topic query within endpoint returns empty array", () => {
+        return request(app)
+          .get("/api/articles?topic=faketopic")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body).toEqual({articles: []})
+          });
+      });
+      test("200: Sort_by query within endpoint returns articles sorted by specific column in default order", () => {
+        return request(app)
+          .get("/api/articles?sort_by=article_id")
+          .expect(200)
+          .then(({ body }) => {
+          
+            expect(body.articles.length).not.toBe(0)
+            const ids = body.articles.map((id) => {
+              return id.article_id
+            })        
+            expect(ids).toBeSorted({ descending: true })
+          })
+        })
+        test("200: Invalid sort by query within endpoint returns default sort (DESC BY DATE)", () => {
+          return request(app)
+            .get("/api/articles?sort_by=not_a_column")
+            .expect(200)
+            .then(({ body }) => {
+      
+              expect(body.articles.length).not.toBe(0)
+              const dates = body.articles.map((article) => {
+                return article.created_at
+              })        
+              expect(dates).toBeSorted({ descending: true })
+            });
+        });
+        test("200: Sort by query followed by an order query modifies the sort order", () => {
+          return request(app)
+            .get("/api/articles?sort_by=article_id&order=ASC")
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.articles.length).not.toBe(0)
+              const ids = body.articles.map((id) => {
+                return id.article_id
+              })        
+              expect(ids).toBeSorted({ ascending: true })
+            });
+        });
+        test("200: Invalid Order query returns articles in default order", () => {
+          return request(app)
+            .get("/api/articles?sort_by=article_id&order=FAKEORDER")
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.articles.length).not.toBe(0)
+              const ids = body.articles.map((id) => {
+                return id.article_id
+              })        
+              expect(ids).toBeSorted({ descending: true })
+            });
+        });
```

Comment:
An addition test for is the topic is valid but there are no articles would be a good test to check here 🧐